include/hx/Anon.h

```diff
@@ -81,10 +81,7 @@ class HXCPP_EXTERN_CLASS_ATTRIBUTES Anon_obj : public hx::Object
       fixed->key = inName;
       fixed->value = inValue;
       if (inValue.type == ::cpp::Variant::typeObject) {
-        HX_OBJ_WB_GET(this, hx::PointerOf(inValue.valObject));
-      }
-      else if (inValue.type == ::cpp::Variant::typeString) {

```

Comment:
Actually, this one should stay.  The value could be a variable, while the name is known to be constant.

Reply:
Fixed.